World.py

```diff
@@ -343,6 +343,9 @@ def __missing__(self, dungeon_name: str) -> EmptyDungeonInfo:
                 if goal_list1 != [goal.name for goal in category.goals] and category.name not in minor_goal_categories:
                     self.one_hint_per_goal = False
 
+        if self.hint_dist_user['one_hint_per_goal']:
```

Comment:
I think this needs to be:

```suggestion
        if 'one_hint_per_goal' in self.hint_dist_user:
```

So setting it to `false` or omitting it entirely behaves as expected.

Reply:
Ah, I see you just made this change.

Reply:
i'll drop a penny in the unit tests caught my bug jar